h/admin/__init__.py

```diff
@@ -9,6 +9,7 @@ def includeme(config):
     config.add_route('admin_badge', '/badge')
     config.add_route('admin_features', '/features')
     config.add_route('admin_cohorts', '/features/cohorts')
+    config.add_route('admin_edit_cohort', '/features/cohorts/{id}')
```

Comment:
That would also make it consistent with the name of the view functions which implement it, which is always nice.